### DISCUSSION TITLE
Permission denied when running on server

### DISCUSSION THREAD

**Discussion by ekpono** [spatie/db-dumper#157] [Q&A]
It works well locally, when I push to production, I get Permission Denied.

```
The dump process failed with exitcode 2 : Misuse of shell builtins : sh: 1: cannot create dump.sql: Permission denied
```

**Comment by tahirkeskin** [comment]
You should check your user permissions. For example, the system user must be authorized in the "dump.sql" folder.

For example; (You have to do the following operation with root user.)

Your System User: ubuntu
Web Server User: www-data
Your Backup Directory: Backups

chown -R ubuntu:www-data backups/

And try again.
